hack/test-gce/fornaxcore_install.sh

```diff
@@ -0,0 +1,105 @@
+#! /bin/bash
+
+set -e
+
+#Golang version
+GO_VERSION=${GO_VERSION:-"1.18.7"}
+
+pushd $HOME
+echo -e "## SETTING UP THE HOSTNAME FORNAXCORE-A\n"
+sudo hostnamectl set-hostname fornaxcore-a
+echo -e "## DISABLING FIREWALL\n"
+sudo ufw disable
+sudo swapoff -a
+
+basic_install() {
+    echo -e "## INSTALL BASIC TOOL"
+    sudo apt-get -y update
+    sudo apt -y install build-essential
+    sudo apt -y install curl
+    sudo apt-get -y install vim
```

Comment:
why you have to install these tools.

Reply:
Brand new vm or instance need these basic tool